MinimedKit/Messages/GetPumpFirmwareVersionMessageBody.swift

```diff
@@ -12,7 +12,7 @@ public class GetPumpFirmwareVersionMessageBody: CarelinkLongMessageBody {
     public let version: String
     
     public required init?(rxData: Data) {
-        let stringEnd = rxData.firstIndex(of: 0) ?? rxData.count
```

Comment:
What is this change for?  I don't see an index(of:) function on Data in the docs.

Reply:
I had a Travis build error otherwise, it could not find firstIndex and suggested replacing it by index to pass the test. But running it locally it ran ok with firstIndex.

Reply:
Ok, that was probably travis using Xcode 9.  You can change this back; we're running Xcode 10.